framework/src/org/checkerframework/common/value/ValueTransfer.java

```diff
@@ -169,18 +169,27 @@ private AnnotationMirror createBooleanAnnotationMirror(List<Boolean> values) {
         AnnotationMirror stringVal = createBooleanAnnotationMirror(resultValues);
         CFValue newResultValue = analysis.createSingleAnnotationValue(
                 stringVal, result.getResultValue().getType()
-                        .getUnderlyingType());
+                                 .getUnderlyingType());
         return new RegularTransferResult<>(newResultValue,
                 result.getRegularStore());
     }
 
     @Override
-    public TransferResult<CFValue, CFStore> visitStringConcatenate(
-            StringConcatenateNode n, TransferInput<CFValue, CFStore> p) {
-        TransferResult<CFValue, CFStore> result = super.visitStringConcatenate(
-                n, p);
-        List<String> lefts = getStringValues(n.getLeftOperand(), p);
-        List<String> rights = getStringValues(n.getRightOperand(), p);
+    public TransferResult<CFValue, CFStore> visitStringConcatenateAssignment(StringConcatenateAssignmentNode n, TransferInput<CFValue, CFStore> p) {
```

Comment:
The method signature for the three methods here are very long; I found the previous line breaks easier to read. (I hate horizontal scrolling in github and all the screen space that is wasted...)